rails-commenting.rb

```diff
@@ -7,24 +7,29 @@
 # FILE: app/controller/blog_posts_controller.rb
 
 # ---1)
+  # Methods can be created in the BlogPostsController to allow users to interact with the blog post database and table. It is a connection between the model and the view in the MVC structure. In websites, the controller provides methods that can be used in the views, but for APIs, there is no need for a view and the controller simply provides the methods to receive requests, retrieve data, and send responses.
 class BlogPostsController < ApplicationController
```

Comment:
What relationship is the `<` showing between BlogPostsController and ApplicationController?